ingest_development_scripts/testing_script.py

```diff
@@ -0,0 +1,43 @@
+from nucleus.sensorfusion import *
```

Comment:
We do this elsewhere in the code but it's generally not good style, so good to avoid if possible